test/admin/e2e/general/e2e.spec.ts

```diff
@@ -424,6 +427,18 @@ describe('General', () => {
       expect(tagName).toBe('a')
     })
 
+    test('should only have one nav item active at a time', async () => {
+      await page.goto(uploadsTwo.list)
+      await openNav(page)
+
+      // Locate all nav items inside the nav group that contain .nav__link-indicator
+      const activeNavItems = page.locator(
```

Comment:
This test should first check that there are two similar hrefs, then ensure only one is active. This will ensure that if that extra collection is removed in the future, this test will fail.